cli/internal/terraform/loader_test.go

```diff
@@ -71,14 +85,14 @@ func TestLoader(t *testing.T) {
 
 			file := file.NewHandler(afero.NewMemMapFs())
 
-			err := prepareWorkspace(file, tc.provider, constants.TerraformWorkingDir)
+			err := prepareWorkspace(tc.action, file, tc.provider, constants.TerraformWorkingDir)
```

Comment:
I would split this up based on action so that 
cluster -> TerraformWorkingDir
iam -> TerraformIAMWorkingDir

Reply:
Is there a typo in the suggested change then? It would make sense to leave the 4th paramter (`constants.TerraformWorkingDir`) out then, right?

Reply:
Already told you this but for the other reviewers: 

`prepareWorkspace` is not bound to TerraformClient specifically, so either both have it hardcoded depending on the action or both should get it supplied as a parameter to keep them roughly in sync.

I think I have a slight preference to pass this as an argument, but as long as it is consistent I don't mind too much.